ui/scss/component/_file-thumbnail.scss

```diff
@@ -0,0 +1,13 @@
+.file-thumbnail__viewed-bar {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 5px;
+  background-color: gray;
+}
+
+.file-thumbnail__viewed-bar-progress {
+  height: 5px;
+  background-color: red;
```

Comment:
@jessopb I wasn't sure if we want (or makes that much sense) to use variables for the colors in this feature.